ehealthid-rp/src/main/resources/i18n_de_DE.properties

```diff
@@ -0,0 +1,17 @@
+lang=de-DE
+title=Anmeldung mit GesundheitsID
+error.login=Einloggen mit GesundheitsID
```

Comment:
is it idiomatic German? @thomasrichner-oviva

Reply:
Let's do a polish pass in a second step, Anna can probably help.